Kiss.Bff.Test/PostKlantContactenCustomProxyTests.cs

```diff
@@ -0,0 +1,445 @@
+﻿using Moq;
+using Microsoft.Extensions.Configuration;
+using System.Text.Json.Nodes;
+using Kiss.Bff.ZaakGerichtWerken.Contactmomenten;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Security.Claims;
+using System.Net;
+using System.Text.Json;
+using Microsoft.Extensions.DependencyInjection;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+
+namespace Kiss.Bff.Test
+{
+    [TestClass]
+    public class PostKlantContactenCustomProxyTests
+    {
+        private Mock<IConfiguration> _configurationMock;
+        private Mock<GetMedewerkerIdentificatie> _getMedewerkerIdentificatieMock;
+        private Mock<IAuthenticationHeaderProvider> _authProviderMock;
+        private PostKlantContactenCustomProxy _controller;
+        private DefaultHttpContext _httpContext;
+
+        [TestInitialize]
+        public void Setup()
+        {
+            _configurationMock = new Mock<IConfiguration>();
+            _getMedewerkerIdentificatieMock = new Mock<GetMedewerkerIdentificatie>();
+            _authProviderMock = new Mock<IAuthenticationHeaderProvider>();
+
+            _configurationMock.Setup(config => config["KLANTCONTACTEN_BASE_URL"]).Returns("https://fakeurl.com");
+
+            var clientHandlerStub = new DelegatingHandlerStub((request, cancellationToken) =>
```

Comment:
wordt al deze logica overal gebruikt? is het misschien netter om per test alleen te mocken wat nodig is?